website/pages/docs/drivers/exec.mdx

```diff
@@ -77,6 +77,16 @@ task "example" {
 }
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| SendSignals | true  |
+| Exec | true |
+| FSIsolation | FSIsolationImage |
+| NetIsolationModes | NetIsolationModeHost, NetIsolationmodeGroup |
```

Comment:
We should include `MustInitiateNetwork | false` everywhere it's not explicitly set to `true` by default.